TACC/Core-Styles
# feat(components): c-nav demo & "--piped" modifier

## Overview / Changes

- Adds demo for `c-nav`.
- Adds `c-nav--piped` variant.

## Related

- documents component thus far only used on https://ecepalliance.org/alliance-members/
- required by https://github.com/TACC/tup-ui/pull/270

## Testing

1. Verify supported variant combinations[^1] are documented.
2. Verify https://ecepalliance.org/alliance-members/ usage (`c-nav c-nav--boxed`) behavior is unchanged.

[^1]: All combinations are supported **except** "piped lists".

## UI

| variant | wide | narrow |
| - | - | - |
| default | ![default wide](https://github.com/TACC/Core-Styles/assets/62723358/2450c87c-626d-4678-87ce-4762062bba7f) | ![default narrow](https://github.com/TACC/Core-Styles/assets/62723358/d618f7d5-d16a-4f02-b29b-9827bd6c1de7) |
| `boxed` | ![boxed wide](https://github.com/TACC/Core-Styles/assets/62723358/7a601281-d00f-4f0a-af8d-b60a40d48cde) | ![boxed narrow](https://github.com/TACC/Core-Styles/assets/62723358/ec5064e4-60a2-4c2e-8f8e-7cf82e0be0c1) |
| `piped` | ![piped wide](https://github.com/TACC/Core-Styles/assets/62723358/785c25d8-bf24-419a-a9b6-81d58078ac09) | ![piped narrow](https://github.com/TACC/Core-Styles/assets/62723358/da64eac8-4a15-4195-9017-d889945c6f2c) |
| `no-list` | ![no-list wide](https://github.com/TACC/Core-Styles/assets/62723358/a793b012-1c38-4a96-a99b-45165327b326) | ![no-list narrow](https://github.com/TACC/Core-Styles/assets/62723358/a5ae6ce0-91f4-4003-a492-e5f8e58c6ca7) |

| readme |
| - |
| ![readme](https://github.com/TACC/Core-Styles/assets/62723358/d073f8ce-8eee-45d0-891c-3d5d0ee3aa95) |
